insets/src/main/java/com/google/accompanist/insets/Insets.kt

```diff
@@ -503,16 +503,26 @@ private fun Insets.updateFrom(windowInsets: WindowInsetsCompat, type: Int) {
     isVisible = windowInsets.isVisible(type)
 }
 
-internal fun Insets.coerceEachDimensionAtLeast(other: Insets): Insets {
-    // Fast path, no need to copy if `this` >= `other`
-    if (left >= other.left && top >= other.top && right >= other.right && bottom >= other.bottom) {
+/**
+ * Ensures that each dimension is not less than corresponding dimension in the
+ * specified [minimumValue].
+ *
+ * @return this if every dimension is greater than or equal to the corresponding
+ * dimension value in [minimumValue], otherwise a copy of this with each dimension coerced with the
+ * corresponding dimension value in [minimumValue].
+ */
+fun Insets.coerceEachDimensionAtLeast(minimumValue: Insets): Insets {
```

Comment:
coerce "each" or "all"?

Reply:
I was 50/50 on this tbh, but leaned towards 'each' to indicate that each dimension is coerced seperately.